relay-dynamic-config/src/defaults.rs

```diff
@@ -25,12 +28,71 @@ pub fn add_span_metrics(project_config: &mut ProjectConfig) {
         return;
     }
 
+    // Add conditions to filter spans if a specific module is enabled.
+    // By default, this will extract all spans.
+    let mut span_op_conditions: Vec<RuleCondition> = Vec::new();
+    let span_op_field_name = "span.op";
+
+    if project_config
+        .features
+        .has(Feature::SpanMetricsExtractionDBModule)
+    {
+        span_op_conditions.push(RuleCondition::And(AndCondition {
+            inner: vec![
+                RuleCondition::Glob(GlobCondition {
+                    name: span_op_field_name.into(),
+                    value: GlobPatterns::new(vec!["db*".into()]),
+                }),
+                RuleCondition::Not(NotCondition {
+                    inner: Box::new(RuleCondition::Eq(EqCondition {
+                        name: span_op_field_name.into(),
+                        value: Value::String("db.redis".into()),
```

Comment:
You can make this a `Value::Array`, in which case the rule will match (or NOT match since you're negating it) if any of the strings in the array match:

```rust
                RuleCondition::Not(NotCondition {
                    inner: Box::new(RuleCondition::Eq(EqCondition {
                        name: span_op_field_name.into(),
                        value: Value::Array(vec!["db.redis".into(), "db.clickhouse".into(), "db.sql.query".into()]),
                        options: Default::default(),
                    })),
                }),
```